sbt-metals/src/main/scala/scala/meta/metals/MetalsPlugin.scala

```diff
@@ -44,7 +44,7 @@ package scala.meta.metals {
             // Disable fatal warnings so that SemanticDBs are generated even for unused warnings.
             // Down the road, metals can even remove unused imports for you if they are reported
             // as warnings but not if they are errors.
-            "-Xfatal-warning"
+            "-Xfatal-warnings"
```

Comment:
This is unrelated to Emacs, I caught it while testing out Metals in the http4s repo. sbt-metals disables fatal warning only when exporting the build to bloop, fatal warnings remain enabled in the sbt build.